### PR TITLE
Allow customizing the timeout on our HTTP calls.

### PR DESCRIPTION
**Pull request by diranged**
@siminm, please review

### CODE REVIEW

File: docs/basic_client.rst [hunk 1]
```diff
@@ -44,7 +44,7 @@ HTTPBin RestConsumer
     class HTTPBinGetThenPost(object):
         def __init__(self, \*args, \**kwargs):
             super(HTTPBinGetThenPost, self).__init__(\*args, \**kwargs)
-            self._api = HTTPBinRestClient()
+            self._api = HTTPBinRestClient(timeout=60)
```
Comment (mikhail) [inline docs/basic_client.rst:47]: What does this change do..?

Comment (diranged) [inline docs/basic_client.rst:47]: Just showing the ability to pass in a custom timeout setting.. the docs still need work here, but right now I was just trying to get this in quickly.

**Comment by mikhail** [comment]
:+1:

**Comment by diranged** [comment]
merged and pushed to pypi

**Pull request merged**